wiki/de/Installation-for-Windows.md

```diff
@@ -75,4 +75,4 @@ Offizielle und ausführliche Infos zur Konfiguration von ASIO4ALL sind auch in d
 
 Jamulus wurde installiert und kann jetzt benutzt werden. Nun schau dir die folgende Seite an:
 
-[Nach der Installation](Getting-Started){: .button}
+[Jamulus setup page](Getting-Started){: .button}
```

Comment:
This needs re-translation.

Reply:
Yes, this is because of changes in EN committed to 'release'. I can give you the translation for Spanish to include it in this PR. You can ask the others for theirs or leave it be - there's not much else you can do.

Spanish: página de configuración de Jamulus

Reply:
Oh, and of course the changes need to be done to the .po files -> then run the create targets script.